Thermal_Expansion/assets/thermalexpansion/textures/gui/Issues.txt

```diff
@@ -0,0 +1,4 @@
+Current issues within GUI textures:
+- "Crush" overlay on igenous extruder not linking to correct texture within CoFH Core mod.
+- JEI Handler png not updated to Soartex, simply upscaled from original
```

Comment:
No **vanilla** textures are allowed to be included either. So if it's not textured it would be simply removed and will fallback to the mods own texture. 

In the case of issues with scaling _(which is very rare)_ the fixed/fanverd texture would need to be created and used.

Reply:
as for this change; the issue is that the texture is not part of the actual GUI image, it is pulled in from the CoFH Core mod textures: the file in question is: "\cofh\textures\gui\elements\scale_compact.png".
For some reason (in my game at least) it is not pulling the Soartex texture, but the vanilla texture.